pkg/logging/logger.go

```diff
@@ -74,18 +81,35 @@ func (l Logger) Error(err error, message string, kvpair ...interface{}) {
 	}
 	mutex.Lock()
 	defer mutex.Unlock()
-	_, found := l.history[err]
-	if found || errors.IsConflict(err) {
-		return
+	le, wrapped := err.(*liberr.Error)
+	if wrapped {
+		err = le.Unwrap()
+		_, found := l.history[err]
+		if found || errors.IsConflict(err) {
+			return
+		}
+		kvpair = append(
+			kvpair,
+			Error,
+			le.Error(),
+			Stack,
+			le.Stack())
+		l.Real.Info(message, kvpair...)
```

Comment:
Would you mind helping me understand why wrapped errors get logged as `INFO`?

Reply:
The only difference is `Logger.Error()` will generate and attach a stack trace which we don't want since we're using the trace captured by the Error.

Reply:
@jortel won't this also mean that if someone turns logging levels down to show only errors (and not info logs), then these wrapped errors won't be logged?

Reply:
The concept of logging levels in Go don't see to follow the conventional approach of (info, error, etc).  As a result, the Logger methods just adds functionality.  The Info() just logs.  The Error() will attach a stack trace etc.  I _think_ the concept of levels is numeric and what the `Logger.V()` method is about.